ubuntu_from_source/Dockerfile

```diff
@@ -0,0 +1,46 @@
+ARG UBUNTU_VERSION=20.04
+
+FROM ubuntu:$UBUNTU_VERSION
+
+LABEL maintainer.name="ROOT team"
+LABEL maintainer.email="root-dev@cern.ch"
+
+ENV LANG=C.UTF-8
+
+COPY packages packages
+
+RUN apt-get update -qq && \
+    ln -sf /usr/share/zoneinfo/UTC /etc/localtime && \
+    apt-get -y install $(cat packages) wget git && \
+    apt-get autoremove -y && \
+    apt-get clean -y && \
+    rm -rf /var/cache/apt/archives/* && \
+    rm -rf /var/lib/apt/lists/*
+
+ENV ROOTSYS /opt/root
+ENV PATH $ROOTSYS/bin:$PATH
+ENV PYTHONPATH $ROOTSYS/lib:$PYTHONPATH
+ENV CLING_STANDARD_PCH none
+
+RUN echo $ROOTSYS/lib >> /etc/ld.so.conf && \
+    ldconfig
+
+RUN yes | unminimize
+
+ARG CMAKE_CXX_STANDARD=17
+ARG ROOT_VERSION=master

```

Comment:
can you please lift these `ARG`s to the top of the file, so they are all together?

Reply:
Okay, I left them there so it doesn't have to rebuild the whole file when you change a single argument, only up to the point it appears.

Reply:
good point, nevermind this comment then